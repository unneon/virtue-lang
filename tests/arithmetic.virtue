a = 2 + 2
b = 123 - 100
c = 6 * 6
d = 100 / 10
print "{a} {b} {c} {d}"

# ignore-llvm
# 4 23 36 10
