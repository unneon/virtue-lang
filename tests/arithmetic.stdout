4 23 36 10
